### ISSUE TITLE
default values are not language specific

### ISSUE THREAD

**Issue by Zen4All** [DivaVocals/zen_MetaTagsImproved#2]
I just had a test drive with this new mod, and I like it.
One major issue in my opinion though, is that "Site Title Default", and "Site Tagline Default" are now configuration values. These can not be set in multiple languages.

**Comment by DivaVocals** [comment]
Yes this is true.. One of my client's requirements was to be able to edit these values from the admin which is why they were moved to be configuration values..

**Comment by DivaVocals** [comment]
Language support it is probably very possible.. Been a while since I looked at this code, but per the readme:
"While the database will include a languages_id, only a single language is supported initially."

I'd welcome any modifications you'd like to submit to better support other languages while maintaining the admin management of the values..
